app/models/manageiq/providers/ansible_tower/shared/automation_manager/job.rb

```diff
@@ -168,6 +168,22 @@ def raw_stdout(format = 'txt')
     raise MiqException::MiqOrchestrationStatusError, err.to_s, err.backtrace
   end
 
+  # Intend to be called by UI to display stdout. The stdout is stored in MiqTask#task_results or #message if error
+  # Since the task_results may contain a large block of data, it is desired to remove the task upon receiving the data
+  def raw_stdout_via_worker(userid, format = 'txt', role = nil)
```

Comment:
This is putting on queue. So better name it something like `raw_stdout_queue` (like in [here](https://github.com/jameswnl/manageiq-providers-ansible_tower/blob/9cc3598ee1affc402f4ded2dcc6e2988ecd00064/app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb#L95-L95))

Reply:
@jameswnl ^ I can do it, but I would change it for both. What does @bzwei think?